src/contractMetadataData.ts

```diff
@@ -2944,7 +2998,7 @@ export const contractMetadataData: ContractMetadataData = [
 		"0x231b0ee14048e9dccd1d247744d114a4eb5e8e63",
 		"ENS: Public Resolver",
 		"ENS",
-		"images/contracts/ENS.jpg",
+		"/images/contracts/ENS.jpg",
```

Comment:
shouldn't the leading slashes be consistent accross? `/images/...` vs `images/...`